framework/behaviors/AttributeBehavior.php

```diff
@@ -85,6 +85,11 @@ class AttributeBehavior extends Behavior
      * @since 2.0.8
      */
     public $skipUpdateOnClean = true;
+    /**
+     * @var bool whether to preserve non-empty attribute values.
+     * @since 2.0.12
+     */
+    public $preserveNonEmptyValues = false;
```

Comment:
@klimov-paul, @samdark, see https://github.com/yiisoft/yii2/pull/13586#discussion_r101737303 for the motivation for $preserveNonEmptyValues